### ISSUE TITLE
A11y_Fluent UI_Fluent UI React v9_Upgrade guide_Ai4W: WCAG 1.3.1Ensures <li> elements are used semantically

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3399]
**GitHubTags:**

#A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG1.3.1.
#Element:listitem; #A11yMAS;#infoandrelationships

**OS Details:**

Web: win11

**Environment Details:**

Application Name: Fluent Ui

**Repro Steps:**

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab navigate till "Upgrade guide" link on the left side bar and hit enter. 
4."Upgrade guide" page will open.
5.Run Ai4W
6.Verify the WCAG 1.3.1Ensures <li> elements are used semantically

**Issue:** Ensures <li> elements are used semantically (listitem - https://accessibilityinsights.io/info-examples/web/listitem)

**Target application:** Upgrade guide - Microsoft Fluent UI - https://react-dev.fluentui.dev/upgrade-guide/

**Element path:** .two-columns.astro-2MWM6TCO:nth-child(24) > .col1.col.astro-2MWM6TCO > .grid.astro-YCT7DGOQ > .feature-list.list > .content.astro-UPXEHOXA > .feature.item.astro-JHBQUSP5:nth-child(2)

**Snippet:** <li class="feature item astro-JHBQUSP5">

**How to fix:** 
Fix any of the following:
  List item does not have a <ul>, <ol> parent element

Environment: Microsoft Edge version 108.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by Latha2564** [comment]
Closing this bug as the bug logged in new ADO
